scripts/cnode-helper-scripts/cntools.library

```diff
@@ -530,52 +531,54 @@ function selectPool {
     pool_dirs+=("${dir}")
   done
   if [[ ${#pool_dirs[@]} -eq 0 ]]; then
-    say "${FG_YELLOW}WARN${NC}: No pools available that can be selected!"
-    [[ ${enc_req_files} -gt 0 ]] && say "* Encrypted pools not listed *"
+    println "ERROR" "${FG_YELLOW}WARN${NC}: No pools available that can be selected!"
+    [[ ${enc_req_files} -gt 0 ]] && println "DEBUG" "* Encrypted pools not listed *"
     return 1
   fi
-  [[ ${enc_req_files} -gt 0 ]] && say "${FG_YELLOW}encrypted pools found but NOT listed, please decrypt to show${NC}"
+  [[ ${enc_req_files} -gt 0 ]] && println "DEBUG" "${FG_YELLOW}encrypted pools found but NOT listed, please decrypt to show${NC}"
   if ! selectDir "pool" "${pool_dirs[@]}"; then return 1; fi # ${dir_name} populated by selectDir function
   pool_name="${dir_name}"
 }
 
 
-# Command    : getPassword [optional:confirm]
-# Description: Get password from user on STDIN
-# Parameters : confirm   >   Optional parameter, force user provide password twise to confirm
-# Return     : populates $password variable, make sure to unset variable when done
-# Examples of Usage:
-#   >> local userPassword=$(getPassword confirm)
+# Command     : getPassword [optional: confirm]
+# Description : Get password from user on STDIN
+# Parameters  : confirm  >  [optional] force user provide password twise to confirm
```

Comment:
`# Parameters  : confirm  >  [optional] force user provide password twice to confirm`